plugins/woocommerce-admin/client/task-lists/progress-header/progress-header.tsx

```diff
@@ -38,25 +37,23 @@ export const ProgressHeader: React.FC< ProgressHeaderProps > = ( {
 			ONBOARDING_STORE_NAME
 		).hasFinishedResolution( 'getTaskList', [ taskListId ] );
 		const visibleTasks = getVisibleTasks( taskList?.tasks || [] );
-
 		return {
 			loading: ! finishedResolution,
 			tasksCount: visibleTasks?.length,
 			completedCount: visibleTasks?.filter( ( task ) => task.isComplete )
 				.length,
-			disabledCompletedCount: visibleTasks?.filter(
-				( task ) => task.isComplete && task.isDisabled
-			).length,
-			hasVisitedTasks:
-				visibleTasks?.filter( ( task ) => task.isVisited ).length > 0,
+			hasFinishedStoreDetails:
+				visibleTasks?.filter(
+					( task ) => task.isComplete && task.id === 'store_details'
+				).length > 0,
 		};
 	} );
 
 	const progressTitle = useMemo( () => {
 		if (
-			( ! hasVisitedTasks &&
-				completedCount < 2 + disabledCompletedCount ) ||
-			completedCount === tasksCount
+			completedCount === 0 ||
+			completedCount === tasksCount ||
+			( hasFinishedStoreDetails && completedCount === 1 )
```

Comment:
This logic doesn't seem to work for me. It always shows `Let's get you started` now at the beginning. Because either the finishedStoreDetails is false and completedCount === 1 or finishedStoreDetails is true and completedCount === 2. Neither of these work for the above condition.

As a simpler fix, could we just filter out the `store_details` task from the previous `hasVisitedTasks` logic?

Reply:
Yes, this is pretty confusing. Sorry for not having been more explicit in my instructions.
The title `Welcome to <store name>` won't be visible if [any task (other than the Store Details) has been finished](https://github.com/woocommerce/woocommerce/issues/32163#issuecomment-1109914307).
If `completedCount` is 1 but the Store Details task is incomplete, the title shouldn't be visible and we will see `Let's get you started 🚀` instead. Same if there is more than one task completed.

On the other hand, although the functionality is almost the same, the change you mentioned makes the code easier to read so I added it.